graphics/fonts/ttf.cpp

```diff
@@ -597,7 +608,11 @@ static void renderGlyph(uint8 *dstPos, const int dstPitch, const uint8 *srcPos,
 				sA = *src;
 
 				uint8 dA, dR, dG, dB;
-				dstFormat.colorToARGB(*rDst, dA, dR, dG, dB);
```

Comment:
Shouldn't this be handled in colorToARGB? This would prevent these changes.

Reply:
Are you suggesting PixelFormat know the transparent color? It might make conceptual sense, but I'd be concerned about performance if always checked there.